.github/workflows/build-rfc-pdf.yml

```
@@ -0,0 +1,60 @@
+            zsh
+          fc-cache -fv
+
+      - name: Install Mermaid CLI
+        run: npm install -g @mermaid-js/mermaid-cli
+
+      - name: Make scripts executable
+        run: chmod +x latex/generator-tex.sh latex/generator-pdf.sh || true
+
+      - name: Generate TeX, PNGs and convert to PDF
+        working-directory: latex
+        shell: zsh
```

actionlint (1.7.7)

shell name "zsh" is invalid. available names are "bash", "pwsh", "python", "sh"

(shell-name)